doc/mdc-migration.md

```diff
@@ -0,0 +1,54 @@
+
+## Changes:
+### Button tag change:
+- One of the primary change with this migration is using `Button` tag, we will start using `com.google.android.material.button.MaterialButton` in views instead of `Button`, `OmegaCenterIconButton`, `PrimarySolidButton`.
+- In order to use icon in a `MaterialButton` use the `icon` attribute instead of `drawableStart` or `drawableEnd` attr.
+- `MaterialButton` also allows us to set `iconGravity` (`textStart`, `viewStart`, `textEnd`, `viewEnd`) for setting the icon position, `iconTint` for setting the icon color.
+  - By default `Clinic.V2.MaterialButton` & other primary button styles for Simple use `iconGravity` of `textStart`, if you want to change that create a new style or override it in the layout file.
+- Material theme will automatically convert any `Button` to `MaterialButton` internally using custom inflater. So if there is a view that extends button the theme values may need to changed if the design specs are broken.
+
+### Style change:
+- `theme_material.xml` will house the main app theme which now extends `Theme.MaterialComponents.Light.NoActionBar`, some of the new attrs we set in app theme.
+```
+<style name="Clinic.V2.Theme" parent="Theme.MaterialComponents.Light.NoActionBar">
+  <item name="materialButtonStyle">@style/Clinic.V2.MaterialButton</item>
+  <item name="materialButtonOutlinedStyle">@style/Clinic.V2.OutlineButton</item>
+  <item name="borderlessButtonStyle">@style/Clinic.V2.TextButton</item>
+  <item name="toolbarStyle">@style/Clinic.V2.ToolbarStyle</item>
+</style>
+```
+These attrs can be directly referenced in the view styles instead of using the entire style tag in the view.
+- We have 4 primary button styles that we use in app
+  - `Clinic.V2.MaterialButton`
+  - `Clinic.V2.TextButton`
+  - `Clinic.V2.OutlineButton`
+  - `Clinic.Button.Flat`
+  - You can then style `MaterialButton` using any of these styles or attr.
+	- Filled Button : `?attr/materialButtonStyle` or `@style/Clinic.V2.MaterialButton`
+	- Text Button: `attr/borderlessButtonStyle` or `@style/Clinic.V2.TextButton`
+	- Outline Button: `?attr/materialButtonOutlinedStyle` or `Clinic.V2.OutlineButton`
+	- Un-Elevated/Flat Button: `@style/Clinic.Button.Flat`
+- You can extend any of those primary button styles to override certain attrs like color, for example:
+```
+<style name="Clinic.V2.MaterialButton.Green3">
+  <item name="backgroundTint">@color/green3</item>
+  <item name="android:textColor">@color/green1</item>
+  <item name="iconTint">@color/green1</item>
+</style>
+```
+
+### Removed widgets:
+- This change will also remove `PrimarySolidButton` & `OmegaCenterIconButton`, we will be using `MaterialButton` in place of that. `MaterialButton` will give us much more control over theming and icon.
+
+## Consequences:
+- Since we started with a design theme, some of the components styles maybe completely or partially broken due this theme switch.
```

Comment:
Can you please expand on this point or reword it to say:
"This is a new design theme that we will be moving to so some of the component styles can be broken either partially or completely because of this theme switch. "